src/Microsoft.Extensions.Configuration.AzureAppConfiguration/EventGridEventParser.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+	/// <summary>
+	/// EventGridEventParser contains the method to parse a Json EventGrid Message Notification
```

Comment:
How about this:
"This class provides a helper method to parse an Event Grid event."